test/functional/fixtures/api/es-next/take-screenshot/test.js

```diff
@@ -119,6 +119,21 @@ describe('[API] t.takeScreenshot()', function () {
                 });
         });
 
+        it('Should check the path argument for forbidden characters', function () {
+            return runTests('./testcafe-fixtures/take-screenshot.js', 'Forbidden characters in the path argument', {
+                shouldFail: true,
+                only:       'chrome'
+            })
+                .catch(function (errs) {
+                    expect(errs[0]).to.contains('There are forbidden characters in the "path:with*forbidden|chars" screenshot path: ":" at index 5 "*" at index 10 "|" at index 20');
```

Comment:
Why do we give `5` as index?
`'path:with*forbidden|chars'.charAt(5)` -> `w`
I think we should give 4

Reply:
I think we can discuss it, but I think that if we use natural language in the error message and print ta problematic path, we should use one-based index, because it's simply faster to find the wrong symbol in the path.

Reply:
I'm not sure. I think testcafe users are programmer and they get used to zero-based index in the strings.
But I don't mind ask other opinions.